src/platform/qt/GBAApp.cpp

```diff
@@ -33,6 +33,7 @@
 #endif
 
 #ifdef BUILD_SDL
+#define SDL_MAIN_HANDLED
```

Comment:
this should be moved to input/SDLInputDriver.h

Reply:
moved